test/add-tests-to-host-test.js

```diff
@@ -2,6 +2,7 @@
 
 const { expect } = require('chai');
 const { createTempDir, buildOutput } = require('broccoli-test-helper');
+const Filter = require('broccoli-persistent-filter');
```

Comment:
Is this dependency only used in tests? Seems like we can use a simpler function to unit test

Reply:
it is. I've been trying to write the test without importing `broccoli-persistent-filter`. but we will still need external addon to read the file, e.g. glob and readFileSync.

Reply:
Updated! @sangm do you mind giving a round of review?